src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Project.json

```diff
@@ -340,7 +340,25 @@
       "Automated": "Yes",
       "MethodName": "CheckBroaderGroupInheritanceSettingsForVarGrp",
       "Rationale": "If a broad group (e.g., Contributors) is configured with excessive permissions at a project level, they are inherited by individual variable groups in the project. Then the integrity of variable groups can be compromised by a malicious user",
-      "Recommendation": "1. Go to Project --> 2. Pipelines -->  3. Library. --> 4. Security. --> 5. Ensure broader groups have read-only access. Refer to detailed scan log (Project.LOG) for broader group list",
+      "Recommendation": "1. Go to Project --> 2. Pipelines -->  3. Library. --> 4. Variable Groups. --> 5. Security. --> 5. Ensure broader groups have read-only access. Refer to detailed scan log (Project.LOG) for broader group list",
+      "Tags": [
+        "SDL",
+        "TCP",
+        "Automated",
+        "AuthZ",
+        "MSW"
+      ],
+      "Enabled": true
+    },    
+    {
+      "ControlID": "ADO_Project_AuthZ_Restrict_Broader_Group_Access_on_Env",
+      "Description": "Do not allow environments to inherit excessive permissions for a broad group of users at project level.",
+      "Id": "Project381",
+      "ControlSeverity": "High",
+      "Automated": "Yes",
+      "MethodName": "CheckBroaderGroupInheritanceSettingsForEnv",
+      "Rationale": "If a broad group (e.g., Contributors) is configured with excessive permissions at a project level, they are inherited by individual variable groups in the project. Then the integrity of variable groups can be compromised by a malicious user",
```

Comment:
Please review the control.json, looks like it is copied from variable group. Please change accrording to the resource type. 

Also get the json reviewed with Manish